app/views/about/accessibility/index.en.html.erb

```diff
@@ -63,22 +57,19 @@
     <h2 class="govuk-heading-m">Compliance status</h2>
     <p class="govuk-body">
       This website is fully compliant with the
-      <a href="https://www.w3.org/TR/WCAG21/" target="_blank" rel="noreferrer noopener">Web Content Accessibility Guidelines version 2.1</a>
+      <a href="https://www.w3.org/TR/WCAG22/" target="_blank" rel="noreferrer noopener">Web Content Accessibility Guidelines version 2.2</a>
       AA standard.
     </p>
 
     <h2 class="govuk-heading-l">Preparation of this accessibility statement</h2>
     <p class="govuk-body">
-      This statement was prepared on 7 June 2024.
-    </p>
-    <p class="govuk-body">
-      This website was last tested on 5 June 2024 against the WCAG 2.1  AA standard.
+      This statement was prepared on 7 June 2024. It was last reviewed on October 2024.
```

Comment:
I think we need to put in the specific date (probably today). "on October" isn't grammatical.